app/main.py

```diff
@@ -1,2 +1,61 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self._capacity = 8
+        self._threshold = int(self._capacity * 2 / 3)
+        self._length = 0
+        self._increase_value = 2
```

Comment:
What is this attribute for? You don't seem to use it.

Reply:
my inaccuracy, tnx)